Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1656,4 +1656,44 @@ NOTE: Restart is required for the setting to take effect.</value>
   <data name="VersionCheck_BuildType_Release" xml:space="preserve">
     <value>release</value>
   </data>
+  <data name="ImportCommand_UnsupportedFileExtensions" xml:space="preserve">
+    <value>The extension of the file '{0}' is not supported.
+
+Import aborted.</value>
+    <comment>{0} filename of unsupported file</comment>
+  </data>
+  <data name="ReplaceProjectContentsFromFilesCommand_DialogCaption" xml:space="preserve">
+    <value>Replace Project Content from Files</value>
+  </data>
+  <data name="ReplaceProjectContentsFromFilesCommand_ConfirmationMessage" xml:space="preserve">
+    <value>Really delete the contents of project '{0}' and replace it with the contents of the selected files?</value>
```

Comment:
I think it should read: `Do you really want to delete the contents of project '{0}' and replace it with the contents of the selected files?`